.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: TOXENV=cover
+      before_install: "travis_retry brew install python augeas dialog"
```

Comment:
GitHub seems to think that this comment is from an outdated diff, however, this still hasn't been addressed. If you disagree with me, please let me know why and we can talk about it, but otherwise, I'd like to see this change made before merging.